src/connectedk8s/azext_connectedk8s/_params.py

```diff
@@ -56,3 +76,12 @@ def load_arguments(self, _):
         c.argument('cluster_name', options_list=['--name', '-n'], id_part='name', help='The name of the connected cluster.')
         c.argument('kube_config', options_list=['--kube-config'], help='Path to the kube config file.')
         c.argument('kube_context', options_list=['--kube-context'], help='Kubconfig context from current machine.')
+
+    with self.argument_context('connectedk8s proxy') as c:
+        c.argument('cluster_name', options_list=['--name', '-n'], id_part='name', help='The name of the connected cluster.')
+        c.argument('token', options_list=['--auth-token'], help='Client authentication token for non-AAD scenario.')
```

Comment:
>non-AAD [](start = 97, length = 7)

we shouldnt call it non-AAD. This works even if cluster is AAD enabled. This is besically used to access the cluster using service account token. Should this be called service-account-token?

Reply:
Yes, I have renamed the scenario to 'service account'. Also renamed the param to --service-account-token.